MoviesMVVM/MoviesMVVM/View/View/MoviesViewController.swift

```diff
@@ -177,22 +185,14 @@ extension FirstViewController: UITableViewDelegate, UITableViewDataSource {
             for: indexPath
         ) as? MovieTableViewCell else { return UITableViewCell() }
 
-        let movie = movieView.cellForRowAt(indexPath: indexPath)
-        cell.setCellWithValues(movie)
+        let movie = moviesViewModel.movies[indexPath.row]
+        cell.setCellWithValues(movie, imageService: moviesViewModel.imageService)
```

Comment:
```suggestion
        cell.setCellWithValues(movie, imageService: moviesViewModel.imageService)
```
ну ты понял по поводу сервиса